fuel-tests/tests/blocks.rs

```diff
@@ -32,6 +32,41 @@ async fn block() {
     assert!(block.is_some());
 }
 
+#[tokio::test]
+async fn produce_block() {
+    let db = Database::default();
+
+    let mut config = Config::local_node();
+
+    config.manual_blocks_enabled = true;
+
+    let srv = FuelService::from_database(db, config).await.unwrap();
+
+    let client = FuelClient::from(srv.bound_address);
+
+    let new_height = client.produce_block(5).await.unwrap();
+
```

Comment:
We should probably strengthen this test to verify that the block height was actually increased. Otherwise this test could continue to pass even if the underlying implementation of the API was completely gutted while returning the expected response.

This might look something like:
 - When I call produce_block with 5,
 - Any transactions I submit afterward are 5 blocks higher than without calling produce_block.

This establishes an expected cause and effect in behavior and also ensures the node continues to function as expected (within reason) after manually creating some blocks.

Reply:
Just added in my tests this logic to ensure the block height actually advances and is represented as such